ui/src/testing/stubs/profile-service-stub.ts

```diff
@@ -0,0 +1,65 @@
+import {Profile} from 'generated';
+import {Observable} from 'rxjs/Observable';
+
+class ProfileStub implements Profile {
+  email: string;
+
+  contactEmail: string;
+
+  enabledInFireCloud: boolean;
+
+  freeTierBillingProjectName: string;
+
+  dataAccessLevel: Profile.DataAccessLevelEnum;
+
+  fullName: string;
+
+  givenName: string;
+
+  familyName: string;
+
+  phoneNumber: string;
+
+  constructor(profile: Profile) {
+    this.email = profile.email;
+    this.contactEmail = profile.contactEmail;
+    this.enabledInFireCloud = profile.enabledInFireCloud;
+    this.freeTierBillingProjectName = profile.freeTierBillingProjectName;
+    this.dataAccessLevel = profile.dataAccessLevel;
+    this.fullName = profile.fullName;
+    this.givenName = profile.givenName;
+    this.familyName = profile.familyName;
+    this.phoneNumber = profile.phoneNumber;
+  }
+}
+
+export class ProfileStubVariables {
+  static PROFILE_STUB = {
+    email: 'testers@researchallofus.org',
+    contactEmail: 'tester@mactesterson.edu',
+    enabledInFireCloud: true,
+    freeTierBillingProjectName: 'all-of-us-free-abcdefg',
+    dataAccessLevel: Profile.DataAccessLevelEnum.Registered,
+    fullName:  'Tester MacTesterson',
```

Comment:
I think it's fine not to use a fuzzer, but please do include some non-latin characters; it's good to verify Unicode handling early and often.

Reply:
Sure, I can do that.

Reply:
One last request -- how about a ☃ for good measure?